front_end/src/app/admin/admin.service.ts

```diff
@@ -5,15 +5,17 @@ import { RequestOptions, URLSearchParams } from '@angular/http';
 import * as moment from 'moment';
 import '../rxjs-operators';
 import { HttpClientService } from '../shared/services';
+import { SetupService } from '../setup/setup.service';
```

Comment:
Can we pull all the stuff on this Admin service out and put it in an app level service.